readiness-tool/popup.js

```diff
@@ -100,6 +100,23 @@ function showSupportedAppsInView(detectedApps, listAllApps) {
     true, listAllApps));
 }
 
+/** @typedef {{ads: !Array<string>, analytics: !Array<string>}} */
+let CategorizedAppsDef;
+
+/** @typedef {{supported: CategorizedAppsDef, notSupported: CategorizedAppsDef}} */
+let FilteredAppsDef;
+
+/** @type {FilteredAppsDef} */
+const filteredApps = {
```

Comment:
Sorry I didn't mean that we should make this a global. I just meant that this is how you would declare a type for `filteredApps`.